package.json

```diff
@@ -0,0 +1,120 @@
+{
+  "name": "@logdna/env-config",
+  "version": "0.0.0",
+  "description": "Configuration package for reading environment variables",
+  "main": "index.js",
+  "scripts": {
+    "test": "tap",
+    "lint": "eslint ./",
+    "pretest": "npm run lint"
+  },
+  "bin": {
+    "config-doc": "./bin/cmd.js"
+  },
+  "keywords": [
+    "env",
+    "env var",
+    "environemnt",
+    "config",
+    "validate",
+    "validator",
+    "validation",
+    "parser"
+  ],
+  "bugs": {
+    "url": "https://github.com/logdna/env-config-node/issues"
+  },
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "url": "https://github.com/logdna/env-config-node"
+  },
+  "homepage": "https://github.com/logdna/env-config-node",
+  "author": "LogDNA, Inc.",
+  "license": "MIT",
+  "eslintConfig": {
+    "root": true,
+    "ignorePatterns": [
+      "node_modules/",
+      "coverage/"
+    ],
+    "extends": [
+      "logdna"
+    ],
+    "parserOptions": {
+      "ecmaVersion": 2019
+    },
+    "rules": {
+      "no-shadow": 0,
+      "no-magic-numbers": 0,
+      "no-new": 0
+    }
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^11.0.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "eslint": "^7.10.0",
+    "eslint-config-logdna": "^1.0.1",
+    "husky": "^4.3.0",
+    "tap": "^14.10.7"
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "commitlint": {
+    "rules": {
```

Comment:
Is there any need to document these rules?

Reply:
If we had a commit template, we could do it there.  That's why I'm thinking both a template and these rules needs to exist together.